cmd/build-push-image/main.go

```diff
@@ -194,13 +200,63 @@ func main() {
 	}
 }
 
+func addNexusBuildArgs(args []string, opts options) (ret_args []string) {
```

Comment:
`ret_args` might be Python style? ;) 

See https://go.dev/doc/effective_go#mixed-caps

There are a few more occurences further below ...

Reply:
I think it would be great to summarise which args are added here in a comment of the function.

Reply:
Changed the python style to mixed caps.
Also added a function comment but not sure whether to duplicate those args which are now already described in the task description.